zio-profiling/jvm/src/main/scala/zio/profiling/causal/ScopeSelectorModule.scala

```diff
@@ -0,0 +1,36 @@
+package zio.profiling.causal
+
+import scala.language.implicitConversions
+
+import zio.profiling.Tag
+
+trait ScopeSelectorModule {
+  import ScopeSelectorModule._
+
+  val *  = Pattern.All
```

Comment:
It might be better not to pollute the global namespace with this, even though the syntax is quite nice.